README.md

```diff
@@ -33,6 +33,8 @@ Copy `dvconfig.py.sample` to `dvconfig.py` (see the `cp` command below) and add
     cp dvconfig.py.sample dvconfig.py
     vi dvconfig.py
 
+Note that the environment variable `$API_TOKEN` will override `api_token` in `dvconfig.py`.
```

Comment:
My only question here is which token should take precedence, the environment variable or the one in dvconfig.py? (I have not looked at this code very deeply overall, so not necessarily advocating for the switch)

Reply:
test

Reply:
test

Reply:
I searched online and it seems standard for environment variable to take precedence.